commands/upload/upload_test.go

```diff
@@ -180,7 +180,7 @@ func TestUploadPropertiesComposition(t *testing.T) {
 	testRunner := func(t *testing.T, test test, verboseVerify bool) {
 		outStream := &bytes.Buffer{}
 		errStream := &bytes.Buffer{}
-		err := runProgramAction(
+		status := runProgramAction(
```

Comment:
`runProgramAction` returns an `error`, we should change `status` to `err` to avoid confusion.